src/examples/assetIssuance.ts

```diff
@@ -0,0 +1,123 @@
+import { BigNumber } from '@polymeshassociation/polymesh-sdk';
+import { Asset } from '@polymeshassociation/polymesh-sdk/types';
+
+import { getClient } from '~/common/client';
+import { parseArgs } from '~/common/utils';
+
+enum SCRIPT_ACTIONS {
```

Comment:
I think this stuff is too distracting in an example file. At the very least this should go at the bottom so its not the first thing people see.

I think its more consistent for the example to do all three actions, without needing extra code to execute single ones. Alternatively this could be 3 different examples.